### ISSUE TITLE
Node Settings Inaccessible From MMN on Identical IP

### ISSUE THREAD

**Issue by Jayto1** [mysteriumnetwork/node#3830]
**Describe the bug**
"Node Settings greyed out in MMN when on the same IP 

**To Reproduce**
1. Go to MMN
2. See error

**Expected behavior**
"Node settings" will be greyed out

**Screenshots**
![image](https://user-images.githubusercontent.com/79473923/129975651-7046cacb-a89a-49f5-83e1-d5f689ffd5a0.png)

**Environment (please complete the following information):**
 - Node version: 0.53.2
 - OS: Raspbian GNU/Linux 10 (buster)
 - WebUI: 0.3.2

**Comment by gantaaman007** [comment]
yes i do have the same issue i am experiencing it from 18-08-21 after upgrading to 0.53.3 version to 0.60.2 . even though in the version 0.53.3 ,it shows the same "you must be on the same local network to access the node settings"

Node version: 0.53.2
OS: windows 10
WebUI: 0.3.2
